torch/nn/modules/module.py

```diff
@@ -433,11 +433,15 @@ def forward(self, x):
     _load_state_dict_post_hooks: Dict[int, Callable]
     _modules: Dict[str, Optional['Module']]
     call_super_init: bool = False
+    _compiled_call_impl : Optional[Callable] = None
```

Comment:
I'm debating whether it also makes sense to save all the args and kwargs to torch.compile so when we load a model we load it with all the settings it was actually compiled with

Reply:
Ok I've decided probably not for now since I don't wanna deal with more torcshcript issues